src/context/settings.tsx

```diff
@@ -165,7 +165,7 @@ export const SettingsProvider: FC<{ children: ReactNode }> = ({ children }) => {
   }, [])
 
   useEffect(() => {
-    if (curEnv === ENVS.PROD) {
+    if (curEnv === ENVS.PROD || curEnv === ENVS.DEV) {
```

Comment:
That line means that 2 of 3 modes use this feature. Does it really make sense to ignore staging?

Maybe this line is not needed anymore?